src/openai/resources/embeddings.py

```diff
@@ -102,7 +103,7 @@ def create(
             "dimensions": dimensions,
             "encoding_format": encoding_format,
         }
-        if not is_given(encoding_format) and has_numpy():
+        if not is_given(encoding_format):
             params["encoding_format"] = "base64"
```

Comment:
I'm not sure if this is always a safe default change to make given the `base64 optimisation isn't enabled for this model yet` comment below. I'm asking around to verify.

(Appreciate that this was basically already the default format as its the default when numpy is installed but this still widens the scope of the default)